### DISCUSSION TITLE
thanks for real world! have a question

### DISCUSSION THREAD

**Discussion by youngsuk-kim** [realworld-apps/realworld#1402] [WIP Implementations]
implement by Kotlin + Mysql + R2DBC + Coroutine + Webflux can this be uploaded in official realworld site? if not, can anybody tell me about restrictions about it?

**Comment by geromegrignon** [comment]
Hi @youngsuk-kim We already have a similar implementation with https://github.com/alphawyu/spring-webflux-mongodb-r2dbc

Changing the database is not enough to provide a real interest here.
So we are left with Coroutine. I made some searches as I have close to no knowledge about Coroutines. I might be wrong but I don't understand how it would be useful for RealWorld given the current specs. Could you provide me with details?


About the restrictions, we avoid too similar implementations.
